runtime/opt/taupage/init.d/10-prepare-disks.py

```diff
@@ -372,17 +386,17 @@ def main():
     else:
         configure_logging(logging.INFO)
 
+    global current_region
```

Comment:
That is just a short cut since region can be overridden from command line and the args are only available in `main()`.  We'll try to come up with a cleaner workaround.